raft/server.cc

```diff
@@ -1469,13 +1471,13 @@ future<read_barrier_reply> server_impl::execute_read_barrier(server_id from, sea
     logger.trace("[{}] execute_read_barrier read id is {} for commit idx {}",
         _id, rid->first, rid->second);
     if (as && as->abort_requested()) {
-        return make_exception_future<read_barrier_reply>(request_aborted());
+        return make_exception_future<read_barrier_reply>(request_aborted("Abort requested while executing read barrier"));
     }
     _reads.push_back({rid->first, rid->second, {}, {}});
     auto read = std::prev(_reads.end());
     if (as) {
         read->abort = as->subscribe([this, read] () noexcept {
-            read->promise.set_exception(request_aborted());
+            read->promise.set_exception(request_aborted("Abort requested while executing read barrier"));
```

Comment:
The message is the same as the previous one - do we want to distinguish these to know the exact place of the abort request?